widget_command.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"fmt"
+	"image"
+	"os/exec"
+	"strings"
+
+	"github.com/muesli/streamdeck"
+)
+
+// CommandWidget is a widget displaying the output of command(s)
+type CommandWidget struct {
+	BaseWidget
+	command string
+	font    string
+}
+
+func NewCommandWidget(bw BaseWidget, opts WidgetConfig) (*CommandWidget, error) {
```

Comment:
Since this never returns any errors, you can simply drop the second return value, and slightly adjust `widgets.go`.